src/Evolution/Systems/RelativisticEuler/Valencia/FixConservatives.hpp

```diff
@@ -0,0 +1,134 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/RelativisticEuler/Valencia/TagsDeclarations.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace RelativisticEuler {
+namespace Valencia {
+
+/*!
+ * \brief Fix conservative variables using the method proposed in F. Foucart's
+ * PhD thesis (Cornell)
+ *
+ * Fix the conservative variables according to the following recipe:
```

Comment:
I suggest being more explicit that this is a pointwise operation. Perhaps "Fix the conservative variables at each grid point according to..."

Reply:
Done.